src/clr-angular/data/datagrid/datagrid.html

```diff
@@ -18,25 +18,30 @@
                 <!--header for datagrid where you can select multiple rows -->
                 <div role="columnheader" class="datagrid-column datagrid-select datagrid-fixed-column"
                      *ngIf="selection.selectionType === SELECTION_TYPE.Multi">
-                            <span class="datagrid-column-title">
-                                <input clrCheckbox type="checkbox" [(ngModel)]="allSelected"
-                                       [attr.aria-label]="commonStrings.selectAll">
-                            </span>
+                  <div class="datagrid-column-wrapper">
+                    <span class="datagrid-column-title">
+                      <input clrCheckbox type="checkbox" [(ngModel)]="allSelected"
+                             [attr.aria-label]="commonStrings.selectAll">
+                    </span>
+                  </div>
                   <div class="datagrid-column-separator"></div>
                 </div>
                 <!-- header for datagrid where you can select one row only -->
                 <div role="columnheader" class="datagrid-column datagrid-select datagrid-fixed-column"
                      *ngIf="selection.selectionType === SELECTION_TYPE.Single">
+                  <div class="datagrid-column-wrapper"></div>
                   <div class="datagrid-column-separator"></div>
                 </div>
                 <!-- header for single row action; only displayType if we have at least one actionable row in datagrid -->
                 <div role="columnheader" class="datagrid-column datagrid-row-actions datagrid-fixed-column"
                      *ngIf="rowActionService.hasActionableRow">
+                  <div class="datagrid-column-wrapper"></div>
```

Comment:
You can't reorder these columns can you?

Reply:
No, they are just static.